src/admin-views/commerce/orders/single/order-items-coupons.php

```diff
@@ -33,5 +35,6 @@
 				</ins>
 			</div>
 		</td>
+		<td></td>
```

Comment:
this seems like a mistake rather than intentional - could use the colspan property instead ?

Reply:
I was following the convention from other templates, but I can change to use a `colspan` instead.